sdk/confidentialledger/Azure.Security.ConfidentialLedger/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "net",
+  "TagPrefix": "net/confidentialledger/Azure.Security.ConfidentialLedger",
+  "Tag": "net/confidentialledger/Azure.Security.ConfidentialLedger_081caadb5d"
```

Comment:
If the prefix here is consistent with the `AssetsRepoPrefixPath`, should we de-dupe it from here?

Reply:
We could, but it's also just keeping it simple to be able to use raw.